pkg/costmodel/cluster.go

```diff
@@ -425,6 +425,23 @@ type nodeIdentifierNoProviderID struct {
 	Name    string
 }
 
+func costTimesMinuteAndCount(activeDataMap map[NodeIdentifier]activeData, costMap map[NodeIdentifier]float64, resourceCountMap map[nodeIdentifierNoProviderID]float64) {
+	for k, v := range activeDataMap {
+		keyNon := nodeIdentifierNoProviderID{
+			Cluster: k.Cluster,
+			Name:    k.Name,
+		}
+		if cost, ok := costMap[k]; ok {
+			minutes := v.minutes
+			count := 1.0
+			if c, ok := resourceCountMap[keyNon]; ok {
+				count = c
+			}
+			costMap[k] = cost * (minutes / 60) * count
+		}
+	}
+}
```

Comment:
Just to be clear, the units here are: `hourly cost per one instance of resource`, `hours` (derived from `minutes`), and `number of instances of the resource`. That sound right?

Reply:
yep!